petab/conditions.py

```diff
@@ -48,8 +48,9 @@ def write_condition_df(df: pd.DataFrame, filename: str) -> None:
         df: PEtab condition table
         filename: Destination file name
     """
+    lint.check_condition_df(df)
     with open(filename, 'w') as fh:
-        df.to_csv(fh, sep='\t', index=True)
+        df.to_csv(fh, sep='\t', index=False)
```

Comment:
Hm. Why would you want to change that? We want conditionId as index.

Reply:
then we need to do something like `index_col=0` in https://github.com/PEtab-dev/PEtab/blob/768eb5b8c145a5de2ee6e054e61120d140de4be8/petab/conditions.py#L25

At the moment the index will be read as "Unnamed" column and have no effect whatsoever.

Reply:
Ah, I see. Okay, I think we want to have `index_col=CONDITION_ID` there.

Reply:
> Ah, I see. Okay, I think we want to have `index_col=CONDITION_ID` there.

That would remove the column from the DataFrame though. And `index=True` would still have no effect.

Reply:
> That would remove the column from the DataFrame though

Yes, that is what is expected everywhere for this moment.